android/src/main/java/fr/greweb/reactnativeviewshot/ViewShot.java

```diff
@@ -368,8 +368,6 @@ private List<View> getAllChildren(@NonNull final View v) {
      */
     private Point captureView(@NonNull final View view, @NonNull final OutputStream os) throws IOException {
         try {
-            DebugViews.longDebug(TAG, DebugViews.logViewHierarchy(this.currentActivity));
```

Comment:
Should I also delete this DebugViews class? seems to be only used here

@gre

Reply:
yes you can remove it 👍

Reply:
Thanks 🙏

Reply:
Cool, deleted it. Feel free to merge.